### ISSUE TITLE
Copy spreadsheet with chart to Closed XML workbook

### ISSUE THREAD

**Issue by vde07010702** [ClosedXML/ClosedXML#1508]
I tried to copy a spreadsheet with a chart made in Excel to a Closed XML workbook. The chart was not copied. Is there a way to copy it?

**Comment by igitur** [comment]
Plead populate the entire issue template.

**Comment by igitur** [comment]
Will reopen when issue template is populated correctly.

**Comment by vde07010702** [comment]
Sorry I did not find the template to complete but I found a solution. Sorry for the post.
Wolfgang
-------- Ursprüngliche Nachricht --------Von: Francois Botha <notifications@github.com> Datum: 03.09.20  13:32  (GMT+01:00) An: ClosedXML/ClosedXML <ClosedXML@noreply.github.com> Cc: vde07010702 <wolfgangsjunkmail@gmail.com>, Author <author@noreply.github.com> Betreff: Re: [ClosedXML/ClosedXML] Copy spreadsheet with chart to Closed XML workbook (#1508) 
Will reopen when issue template is populated correctly.
